Pages/FoodPages/Create.cshtml.cs

```diff
@@ -37,7 +37,7 @@ public IActionResult OnGet()
         public Food Food { get; set; }
         [BindProperty]
         [Display(Name = "Image")]
-        [Required(ErrorMessage = "Pick an Image")]
+        //[Required(ErrorMessage = "Pick an Image")]
```

Comment:
Why was this commented? Curious

Reply:
Just for not to upload image every time I create new food item